source/guides/git.rst

```diff
@@ -14,153 +14,117 @@ For example, you're trying to develop a feature on Scilpy and your collegue is d
+    - *conflict*: A situation that occurs when two or more branches have conflicting changes to the same part of a file, requiring manual resolution.
+    - *delta*: The difference between two sets of file contents, often used to represent changes between commits or branches.
 
-.. figure:: /images/intro_to_git_tree.png
-   :scale: 80 %
+.. figure:: /images/intro_to_git_organization.png
+   :scale: 25 %
    :align: right
 
-Each update is called a **commit**. This is shown in the second figure [2]_.
-
-.. code-block:: bash
-
-    # From the workspace to your local repo:
-    git status # To see which files have been modified without telling git
-    git add my_file  # To add or update a file in git's memory.
-    git add -u      # To add all files that have been modified, but that git already knew
-    git status      # If you check again, everything in green will be recorded with the next command:
-    git commit -m "Message to explain the work I have been doing in this update"
+Here is a figure [1]_ with a summary of the commands that might be useful, but you can google for more information. Every action happening inside the cloud (on internet) isn't performed in command line, but with buttons on Github. Inversely, every action concerning your local computer is performed in command-line.
 
-    # From the local repo to your forked repo
-    git push origin my_branch  # Will automatically send it to the same branch name in your forked repo.
+In Git, your working directory is the local folder containing the current state of your project files. As you make changes to these files, they are tracked by Git but not yet committed (see :bash:`git status`). 
 
-    # From your forked repo to the upstream repo:
-    # Use the Pull Request button on internet
+The staging area, also known as the index, is an intermediate space where you can organize and prepare the changes you want to include in your next commit (see :bash:`git add`). By adding changes to the staging area, you're essentially telling Git which changes should be recorded in the upcoming commit (see :bash:`git commit`). Your local repository is the hidden .git folder within your working directory, where Git stores the entire version history, including all commits and branches. It is separate from the working directory, ensuring that your project's history is preserved even if you make changes or delete files in your working directory.
 
-    # From your local repo to the upstream repo:
-    # DON'T DO THAT. Didn't you see that there is no such arrow on Figure 1!
+A remote repository is a version of your project stored on a remote server, allowing for collaboration with other developers. You can push your local commits (see :bash:`git push`) to the remote repository and pull changes made by others (see :bash:`git pull` or :bash:`git fetch`), keeping your project in sync across multiple environments.
```

Comment:
I feel like this should be said earlier, maybe it would fit where commits and pushes are explained.